Makefile

```diff
@@ -43,4 +43,8 @@ cmd/ui/dist:
 
 npm-build:
 	$(MAKE) -C ui/ build
-.PHONY: npm-build
\ No newline at end of file
+.PHONY: npm-build
+
+update-openfga-model:
+	@printf 'package openfga\n\n// Code generated by Makefile; DO NOT EDIT.\n\nvar authModel = `%s`\n' '$(shell npx @openfga/syntax-transformer transform --from=dsl --inputFile=./internal/openfga/schema.openfga | jq -c)' > ./internal/openfga/auth_model.go
```

Comment:
Hey! I'm the product manager for OpenFGA and found this PR :)

You can do this using the [OpenFGA CLI](https://github.com/openfga/cli) using `fga model transform ./internal/openfga/schema.openfga`, and actually you don't need the JSON representation anymore, you can just write the `.openfga` model directly to a store using `fga model write --store-id=.. --file schema.openfga` without having code generated by the makefile.

Reply:
Hello! Thanks a lot for the suggestions.

This is used for transforming the `.openfga` model to a go object. It is a hacky approach and we will probably have to revisit it.